static/app/views/alerts/index.spec.tsx

```diff
@@ -1,8 +1,14 @@
 import {render, screen} from 'sentry-test/reactTestingLibrary';
 
+import TeamStore from 'sentry/stores/teamStore';
 import AlertsContainer from 'sentry/views/alerts';
 
 describe('AlertsContainer', function () {
+  beforeEach(() => {
```

Comment:
The reason for the unrelated change from here is that all these components use the component I tough and when writing tests they just assumed everyone has at least one team.